src/SprykerSdk/Spryk/Model/Spryk/Builder/Template/TemplateSpryk.php

```diff
@@ -182,4 +202,44 @@ protected function getFilenameFromTemplateName(SprykDefinitionInterface $sprykDe
 
         return $filename;
     }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface $sprykDefinition
+     */
+    protected function checkCoreExtensionToProjectLevel(SprykDefinitionInterface $sprykDefinition): void
```

Comment:
You should return the `SprykDefinitionInterface` to show that it might be changed inside this class.